Userland/Libraries/LibGUI/Notification.h

```diff
@@ -30,6 +30,7 @@
 #include <LibGfx/Bitmap.h>
 
 namespace GUI {
+    class NotificationServerConnection;
```

Comment:
Formatting: No leading spaces, and while you're add it please add a newline between `namespace GUI {` and `class NotificationServerConnection;`.